libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -74,6 +74,9 @@ class AP_Motors {
     // return string corresponding to frame_type
     virtual const char* get_type_string() const { return ""; }
 
+    // returns a formatted string into buffer, e.g. "QUAD/X"
+    void get_frame_and_type_string(char *buffer, uint8_t buflen);
```

Comment:
looks pretty good.. i wonder if get_frame_and_type_string can be made const.  not the arguments but the method itself.

Reply:
Yes, it can.  It was already `const` in the body of the function - I managed to think it was already marked as `const` in the header, which confused me for a moment last night but I moved on...

I've now marked it `const` in the header.